includes/today-options-gmucf.php

```diff
@@ -18,6 +19,306 @@ function tu_add_gmucf_options_page() {
 	}
 }
 
+/**
+ * Function for adding the GMUCF Options Page
+ * ACF field groups & fields
+ **/
+function tu_add_gmucf_options_page_fields() {
+	if ( function_exists( 'acf_add_local_field_group' ) ) {
```

Comment:
It would probably be okay to just do a negative test and return out. That way the rest of the code block doesn't have to be indented in.

Reply:
Updated